### ISSUE TITLE
Examples in B-U585I-IOT02A_BSP@1.0.0 pack no longer compatible with Keil::STM32U5xx_DFP@2.0.0 pack

### ISSUE THREAD

**Issue by edriouk** [MDK-Packs/Pack#4]
Examples in the BSP use gpdsc location defined by pack Keil::STM32U5xx_DFP@1.2.0 : 
```
      <gpdsc name="$PSTM32CubeMX/FrameworkCubeMX.gpdsc"/>        <!-- path is either absolute or relative to working directory -->
      <workingDir>$PSTM32CubeMX</workingDir>                     <!-- path is specified either absolute or relative to PDSC or GPDSC file -->
```
From Keil::STM32U5xx_DFP@2.0.0 another location is used:
```
    <gpdsc name="FrameworkCubeMX.gpdsc"/>                         <!-- path is either absolute or relative to working directory -->
    <workingDir>$PBoard/$B/STM32CubeMX</workingDir>               <!-- path is specified either absolute or relative to PDSC or GPDSC file -->
  ```
This causes that some tools cannot find required gpdsc anymore. 
I believe another version of the pack with corrected examples should be created.

**Comment by ZigaMahne** [comment]
I have investigated described issue and try to duplicate it.

The conclusion is that Keil.B-U585I-IOT02A_BSP@1.0.0 pack is synhroized with Keil.STM32U5xx_DFP@2.0.0.

Examples in Keil.B-U585I-IOT02A_BSP@1.0.0 use gpdsc location defined by pack Keil.STM32U5xx_DFP@2.0.0 
( `$PBoard/$B/STM32CubeMX` from "Keil::Device.STM32Cube Framework.STM32CubeMX" component  version 2.0.0 ).

With @edriouk  we reinstall Keil.B-U585I-IOT02A_BSP@1.0.0 pack on his enviroment and issue was resolved.
